modules/manage/pages/manage-backup-and-restore/manage-backup-and-restore.adoc

```diff
@@ -0,0 +1,799 @@
+
+Alternatively scheduled merges and backups can be configured by accessing the control at the left-hand side of the *Period* field:
+
+image:manage-backup-restore/scheduleOptionsMenu.png[,420,align=left]
+
+The pull-down menu thus displayed contains three kinds of scheduling option.
+One is the default, *Simplified*.
+Another is by means of _time-units_: which are *Minutes*, *Hours*, *Days*, and *Weeks*.
+If a unit is specified, an appropriate integer must be entered into the *Frequency* field, to indicate the number of time-units that must elapse between repetitions of the task.
+If *Minutes* are specified, only the minutes portion of the time entered into the *Time* field is used &#8212; as the starting point, every hour, for the task sequence; with the task being repeated throughout the hour as many times as specified in the *Frequency* field.
+If *Hours* are specified, the task is first performed at the time specified in the *Time* field, and then repeated as specified in the *Frequency* field.
+If *Days* or *Weeks* are specified, the task is performed as scheduled, at the time specified in the *Time* field.
+(A full example of using *Minutes* as time-units is provided above, in xref:manage:manage-backup-and-restore/manage-backup-and-restore.adoc#schedule-backups[Schedule Backups].)
+
+A third option is by means of _days_: such as *Monday*, *Tuesday*, and so on.
+If a day is selected, an appropriate integer must be entered into the *Frequency* field, to indicate the number of instances of the day's occurrence that should elapse before the task is performed.
+For instance, if *Monday* and *3* are specified, the task is performed on the first Monday, then two Mondays elapse, and then the task is performed again on the fourth Monday; and so on.
+The time at which the task is commenced is that specified in the *Time* field.
+
+[#use-cloud-storage]
+== Use Cloud Storage
+
+During the input-sequence for repository-creation, storage for the repository can be specified.
+See xref:manage:manage-backup-and-restore/manage-backup-and-restore.adoc#schedule-backups[Schedule Backups], for an example of using the local filesystem.
+Alternatively, cloud storage can be selected, from the *Storage Locations* field of the *Create Repository* dialog.
+When the selection is made, the dialog expands vertically, as follows:
+
+image:manage-backup-restore/cloudStorageOptions.png[,420,align=left]
+
+The fields are as follows:
+
+* *Staging Directory*.
+A directory on the local file system that is large enough to accommodate approximately 10% of the data set that is to be backed up.
+A minimum of 50 GB is recommended.
+This location must be accessible by all nodes in the cluster.
+
+* *Provider*.
+To use either S3 or S3-compatible storage, select *AWS*.
+
+* *Cloud Bucket*.
+The bucket in the cloud to which data will be backed up.
+
+* *Path Prefix*.
+The path of the archive, within the cloud bucket.
+This should take the form `s3://${BUCKET_NAME}/${ARCHIVE_PATH}`.
```

Comment:
path prefix should just be of the format `/path/inside/the/bucket`

Reply:
I've changed it to say: This should take the form `/path/inside/the/bucket`.

Reply:
At https://github.com/couchbase/docs-server/pull/1596/files